index.bs

```diff
@@ -1281,8 +1289,10 @@ partial interface HTMLIFrameElement {
         ::  null
         :   [=PermissionsPolicyViolationReportBody/disposition=]
         ::  |disposition|
-        :   [=PermissionsPolicyViolationReportBody/allowAttribute=]
+        :   [==]
```

Comment:
This one disappeared with the latest commit, I think.

Reply:
Fixed, sorry about that :(